### PR TITLE
Add a blog post announcing the release of Ember Data 1.0.0-beta.15

### PR DESCRIPTION
**Pull request by bmac**
cc @igorT @fivetanley @wecc

### CODE REVIEW

File: source/blog/2015-02-14-ember-data-1-0-beta-15-released.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+```
+
+
+Please give a warm thanks to [Christoffer Persson][wecc] for all his
+work implementing this feature!
+
+### Errors on arbitrary properties
+
+Previously, Ember Data would only attach errors to a record's error
+object if they matched a property an property that was already on the
```
Comment (wecc) [inline source/blog/2015-02-14-ember-data-1-0-beta-15-released.md]: Remove "an property"?

File: source/blog/2015-02-14-ember-data-1-0-beta-15-released.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+record. Now thanks to [Alex Speller][alexspeller]'s
+[work](https://github.com/emberjs/data/pull/1984) any error returned
+by the backend can be attached to the error object.
+
+Please note that if you backend is returning an error property that is
```
Comment (wecc) [inline source/blog/2015-02-14-ember-data-1-0-beta-15-released.md]: you => your

File: source/blog/2015-02-14-ember-data-1-0-beta-15-released.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+not an attribute on the record you will need to manually clear it
+before the record can transition out of the error state.
+
+### Sort query params in ajax calls.
+
+When developing APIs, is pretty common to use a caching mechanism,
```
Comment (wecc) [inline source/blog/2015-02-14-ember-data-1-0-beta-15-released.md]: is => it's ?

File: source/blog/2015-02-14-ember-data-1-0-beta-15-released.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+tools use the URL string to determine if they have a cached response.
+
+As an example, if one user requests `/posts?sort=price&category=pets`
+and another requests `/posts?category=pets&sort=price`, the cached
+request won't be used in the second call.
+
+Ember Data's `RESTAdapter` will now sort the query parameters by their
+field name when making an ajax request.
+
+It is possible to disable this behavior, by setting setting
```
Comment (wecc) [inline source/blog/2015-02-14-ember-data-1-0-beta-15-released.md]: double "setting"

**Comment by bmac** [comment]
Thanks @wecc. I've update this pr with your corrections.

**Comment by wecc** [comment]
LGTM :+1:

**Pull request merged**